openpype/hosts/traypublisher/plugins/publish/collect_simple_instances.py

```diff
@@ -95,6 +104,40 @@ def process(self, instance):
             )
         )
 
+        # Add targeted family to families
```

Comment:
This whole content should be in different plugin. This would work only for simple creators.

Reply:
aren't all creators of Traypublisher simple creators?

Reply:
No, there are by default editorial creators, online creator and we have many custom creators specific for client workflows.

Reply:
still not following, this plugin is only running for traypublisher host? what's the difference of this and the `collect_writes` of Nuke for example? That's doing the same and not in a different plugin?

Reply:
SimpleCreator is not the only creator that can be used in Traypublisher. We have other creators for "traypuslisher" host that create the same families as simple creators do, but with more sofisticated logic.

Reply:
exactly my point, if they aren't going through this plugin, why is it a worry that this plugin does this?